safe-global/safe-wallet-web
# fix: Keep Sidebar drawer mounted

## What it solves

The Sidebar has a delay when opening with a lot of safes. This PR keeps the Sidebar Drawer mounted to resolve it. It does not refactor the hooks that are slow.

## How this PR fixes it

- Adds `keepMounted: true` to the Sidebar Drawer

## How to test it

1. Open a Safe with a wallet that owns a lot of safes
2. Open and close the sidebar a few times
3. Observe no delay when doing so

## Screenshots

## Checklist
